src/server/github_handler.rs

```diff
@@ -710,13 +710,15 @@ impl GithubEventHandler {
                                     .build(),
                             );
 
+                        let commits = self.pull_request_commits(&pull_request);
+
                         self.messenger.send_to_all(
                             &message,
                             &attachments,
                             &pull_request.user,
                             &self.data.sender,
                             &self.data.repository,
-                            &self.all_participants(&pull_request),
+                            &self.all_participants_with_commits(&pull_request, &commits),
```

Comment:
this was already doing `self.pull_request_commits` internally. this makes it only do this lookup once, and allows us to re-use for `check_jira_refs` below.